docs/assemblies/proc_hotfixing-the-data-plane-container-content-rpms.adoc

```diff
@@ -0,0 +1,46 @@
+[id="proc_hotfixing-the-data-plane-container-content-rpms-{context}"]
+= Hotfixing the data plane container content with RPM's
+
+[role="_abstract"]
+
+When container hotfix content is delivered as RPM's, container images must be
+updated manually.
+
+.Procedure
+
+. From a RHEL workstation, server, or virtual machine, ensure the following packages are installed
++
+* buildah
+* podman
+
+. From a RHEL workstation, server, or virtual machine, collect the hotfix RPMs into a new directory
+----
+$ mkdir -p <hotfix_id>/rpms
+$ cp /path/to/hotfix/*.rpm <hotfix_id>/rpms
+----
```

Comment:
```suggestion
+
----
$ mkdir -p <hotfix_id>/rpms
$ cp /path/to/hotfix/*.rpm <hotfix_id>/rpms
----
```
This might be where the + is needed @fao89